Statusseite: ImageName, Mesh-MAC, Tunnel-MAC und Sitecode.

Fuer Gluon 2025.1 neu geschrieben. Zwei Teile des alten Patches sind
entfallen, weil Upstream sie inzwischen selbst zeigt:

  * Target/Subtarget - steht dort als "target (subtarget)" statt
    "target/subtarget". Das Format ist in Ordnung, die Zeile ist doppelt.
  * Gluon Version - steckt in der Firmware-Zeile als "release (base)".

Die Tunnel-MAC haengt jetzt als eigener if-Block hinter dem
vpn_pubkey-Block, statt dessen Verschachtelung umzubauen. 2025.1 hat den
mesh_vpn-Bereich um den public key erweitert; ein eigener Block ist
robuster gegen weitere Aenderungen dort.

--- a/package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html
+++ b/package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html
@@ -165,7 +165,9 @@
 						</a></dd>
 					<%- end %>
 					<dt><%:Model%></dt><dd><%| nodeinfo.hardware.model %> (<%| nodeinfo.hardware.nproc %> CPUs)</dd>
+					<dt><%:ImageName%></dt><dd><%| nodeinfo.software.firmware.image_name %></dd>
 					<dt><%:Primary MAC address%></dt><dd><%| nodeinfo.network.mac %></dd>
+					<dt><%:Mesh MAC-Adresse%></dt><dd><%| nodeinfo.network.mesh.bat0.interfaces.other[1] %></dd>
 					<dt><%:IP address%></dt><dd><%= pcdata(table.concat(sorted(nodeinfo.network.addresses), '\n')):gsub('\n', '<br>') %></dd>
 					<dt><%:Firmware%></dt><dd><%| nodeinfo.software.firmware.release %> (<%| nodeinfo.software.firmware.base %>)</dd>
 					<dd><%| nodeinfo.software.firmware.target %> (<%| nodeinfo.software.firmware.subtarget %>)</dd>
@@ -179,6 +181,9 @@
 							<% if vpn_pubkey then -%>
 							<br><%| vpn_pubkey %> (public key)
 							<%- end %>
+							<% if nodeinfo.network.mesh.bat0.interfaces.tunnel and nodeinfo.network.mesh.bat0.interfaces.tunnel[1] then -%>
+								<br>mac: <%| nodeinfo.network.mesh.bat0.interfaces.tunnel[1] %>
+							<%- end %>
 						</dd>
 						<% if nodeinfo.network.mesh_vpn.bandwidth_limit.enabled then -%>
 						<dt><%:Bandwidth limit%></dt>
@@ -192,6 +197,7 @@
 						</dd>
 						<%- end %>
 					<%- end %>
+					<dt><%:Sitecode%></dt><dd><%| nodeinfo.system.site_code %></dd>
 					<dt><%:Site%></dt><dd><%| site.site_name() %></dd>
 					<% if nodeinfo.system.domain_code then -%>
 						<dt><%:Domain%></dt>
